pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientBuilderImpl.java

```diff
@@ -138,6 +139,24 @@ public ClientBuilder authentication(String authPluginClassName, Map<String, Stri
         return this;
     }
 
+    private void setAuthenticationFromPropsIfAvailable(ClientConfigurationData clientConfig) {
```

Comment:
This change is NOT only about the "Admin" client but it is also touching the regular Client.

Please update the title and the description